docs/xks/developer-guide/security.md

```diff
@@ -9,3 +9,19 @@ Certain policies will always be enforced in the cluster as a guard rail to minim
 any request to create or update a Kubernetes resource will first have to pass through the policy check. If the resource in question does not comform to the policy it will be rejected
 by the API server when applying the change. Knowing this is important as certain features or options documented on the Internet may not be available or are restricted in the XKS service.
 This can include things like certain types of volume mounts, labeling requirements or container capabilities for a Pod.
+## Security Features
+### OPA Gatekeeper
+
+OPA Gatekeeper is a product in which we have a set of rules defining how we allow containers to be created. To simplify this for the developers these rules are then applied automatically to all pods created to achive our set standard. 
```

Comment:
Add some links to what OPA is and write out the abbreviation. 
Also link to: https://github.com/XenitAB/gatekeeper-library to show where these rules comes from.